src/complex.h

```diff
@@ -194,6 +194,34 @@ class Complex : public Number {
             return from_mpq((this->real_ * other.i) / conjugate, (this->imaginary_ * (-other.i)) / conjugate);
         }
     }
+
+    /*! Divide other by the Complex
+     * \param other of type Rational
+     * */
+    inline RCP<const Number> rdivcomp(const Rational &other) const {
+        mpq_class conjugate = this->real_*this->real_ + this->imaginary_*this->imaginary_;
+        if (conjugate.get_num() == 0) {
+            throw std::runtime_error("Divide by zero.");
+        } else {
+            return from_mpq((this->real_ * other.i) / conjugate, (this->imaginary_ * (-other.i)) / conjugate);
+        }
+    }
+
+    /*! Divide other by the Complex
+     * \param other of type Complex
+     * */
+     inline RCP<const Number> rdivcomp(const Complex &other) const {
```

Comment:
We don't really need this method because of `divcomp(const Complex &other) const {` method of `Complex`